api/src/main/kotlin/team/backend/presentation/dto/QuizDto.kt

```diff
@@ -0,0 +1,19 @@
+package team.backend.presentation.dto
+
+import jakarta.validation.constraints.NotEmpty
+import jakarta.validation.constraints.NotNull
+
+class QuizDto {
+
+    class AnswerRequest(
```

Comment:
@mangchhe Is there a reason why you used regular class instead of data class?

Reply:
@BAEKJungHo 
There is no particular reason.
I think the data class is suitable, so I will change it.